gpkit/interactive/plot_sweep1d.py

```diff
@@ -0,0 +1,48 @@
+"Implements plot_sweep1d function"
+import matplotlib.pyplot as plt
+import numpy as np
+from ..exceptions import InvalidGPConstraint
+from ..tools import autosweep_1d
+
```

Comment:
I'd have `sol = Model.autosweep("this_variable", minval, maxval, tol)` and then `sol.plot_sweep()`, which would automatically infer the swept variable or use an optional argument in case of multiple.